ini/stm32f4.ini

```diff
@@ -75,7 +75,7 @@ extra_scripts       = ${common.extra_scripts}
 platform          = ${common_stm32f1.platform}
 extends           = common_stm32f1
 board             = genericSTM32F103RC
-board_build.address  = 0x08005000
+board_upload.offset_address  = 0x08005000
 board_build.ldscript = fly_mini.ld
 extra_scripts     = ${common_stm32f1.extra_scripts}
   buildroot/share/PlatformIO/scripts/custom_board.py
```

Comment:
I just noticed this and am wondering why there's an STM32F1 based board in the STM32F4 file.... :thinking: